### PR TITLE
Moved to Styles.css

### PR DESCRIPTION
**Pull request by JipSterk**
@ufocoder is this what you where looking for?
or would you like a seperate .css file for each component that needs styling?

### CODE REVIEW

**Comment by ufocoder** [comment]
As variant it could be like your solution

**Comment by JipSterk** [comment]
@ufocoder i don't know why codeclimate keeps failing.
Any suggestions?

**Comment by ufocoder** [comment]
@JipSterk I'm still recommending you to adhere to this structure:
```
[component]:
 - index.js[x]
 - styles.css
 - image.png
```

Something like that I've described it the issue #9. Check it

**Comment by ducthienbui97** [comment]
There's conflict, can you update your repo?

File: src/components/User/styles.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  margin-top: 1em;
+  border-radius: 5px;
+  font-size: 1.2em;
+  padding-top: 0.5em;
+  padding-bottom: 0.5em;
+}
+
+.row {
+  margin-right: 0px;
+  margin-left: 0px;
```
Comment (ducthienbui97) [inline src/components/User/styles.css]: Codeclimate is complaining "Values of 0 shouldn't have units specified."

**Review by ducthienbui97 — CHANGES_REQUESTED** [review]

**Comment by ducthienbui97** [comment]
Most of the complains were already there before.

**Review by ducthienbui97 — APPROVED** [review]

**Pull request merged**